contributing.md

```diff
@@ -139,7 +142,23 @@ The TSC decides whether to have 1 or 3 chairs. They:
 - May be non-developers
 - Cannot all be from the same company (if 3 co-chairs)
 
-> Current TSC Chairs are **interim** until **June 2025**, after which elections will be held.
+#### Nominations
+
+- Nominations for TSC Chair(s) are open to all Organizers
+- Interested persons may nominate themselves
+- Nominations are facilitated by the Linux Foundation ([example](https://github.com/orgs/fairpm/discussions/43))
+
+#### Term Limits and Rotation
+
+To ensure continuity and stable leadership, Co-Chair terms are staggered as follows:
+
+* All Co-Chair positions serve a three-year term
+* To establish staggered terms:
+  * The initial three Co-Chairs will serve 1-, 2-, and 3-year terms, as agreed upon among themselves
+  * After the first year, one Co-Chair will rotate off and a new one will be elected annually
+* This rotation ensures:
```

Comment:
Maybe "This rotation helps preserve stability:"

Reply:
Added in [this commit](https://github.com/fairpm/tsc/pull/46/commits/8407d89e0a4050449d0b332001873d8ab5faf730)